LambdaEngine/Source/Game/ECS/Systems/Rendering/AnimationSystem.cpp

```diff
@@ -55,13 +78,16 @@ namespace LambdaEngine
 			// Interpolate position
 			glm::vec3 position;
 			{
+				// If the clip is looping the last frame is redundant
+				const uint32 NumPositions = animation.IsLooping ? channel.Positions.GetSize() - 1 : channel.Positions.GetSize();
```

Comment:
Should be lower case. Same with `NumRotations`.
```suggestion
				const uint32 numPositions = animation.IsLooping ? channel.Positions.GetSize() - 1 : channel.Positions.GetSize();
```

Reply:
Yep, my bad 👀